nsxt/policy_common.go

```diff
@@ -395,6 +395,44 @@ func getPolicySecurityPolicySchema(isIds, withContext, withRule bool) map[string
 	return result
 }
 
+func getIgnoreTagsSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:     schema.TypeList,
+		Optional: true,
+		Computed: true,
+		Elem: &schema.Resource{
+			Schema: map[string]*schema.Schema{
+				"scopes": {
+					Type:        schema.TypeList,
+					MaxItems:    1,
+					Description: "List of scopes to ignore",
+					Required:    true,
+					Elem: &schema.Schema{
+						Type: schema.TypeString,
+					},
+				},
+				"detected": {
+					Type:        schema.TypeSet,
+					Description: "Tags matching scopes to ignore",
+					Computed:    true,
+					Elem: &schema.Resource{
+						Schema: map[string]*schema.Schema{
```

Comment:
Can `getTagsSchemaInternal()` be recycled here in some way?
Other than this LGTM

Reply:
This one is fully computed, I didn't want to add to many params to `getTagsSchemaInternal`